Google.Cloud.AspNetCore/Google.Cloud.AspNetCore.IntegrationTests/FirestoreCache/FirestoreCacheTest.cs

```diff
@@ -0,0 +1,195 @@
+﻿// Copyright (c) 2019 Google LLC.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Google.Cloud.ClientTesting;
+using Google.Cloud.Firestore;
+using Microsoft.Extensions.Caching.Distributed;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Google.Cloud.AspNetCore.Firestore.DistributedCache
+{
+    public class FirestoreCacheTest : IClassFixture<FirestoreCacheTestFixture>
+    {
+        private readonly FirestoreCacheTestFixture _fixture;
+        string _key = Guid.NewGuid().ToString();

```

Comment:
Nit: explicitly private for both this field and the next

Reply:
Done.